src/core/Button/Button.baseStyles.tsx

```diff
@@ -64,25 +64,35 @@ const secondaryStyles = css`
 `;
 
 const secondaryNoBorderStyles = css`
-  &.fi-button--secondary-noborder {
+  &.fi-button--secondaryNoBorder {
```

Comment:
this should probably be `fi-button--secondary-no-border` or `--secondary-noborder`

Reply:
Seems to be fixed!